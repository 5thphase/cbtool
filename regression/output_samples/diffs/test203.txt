|simzone_X |5 |sut |simzone_X |xxx.xxx.xxx.xxx 
|simzone_X |5 |lg |simzone_X |xxx.xxx.xxx.xxx 
|Redis Version |2.8.4 
