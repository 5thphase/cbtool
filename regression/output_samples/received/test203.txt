 status: VMC E65CBBAE-AC7D-5374-83BA-971BC910809C was successfully registered on SimCloud "TESTCLOUD".
 status: VMC 22FD0315-002B-5D68-8C4F-88C6FE47FD88 was successfully registered on SimCloud "TESTCLOUD".
 status: VMC F3070B0B-F942-580F-996F-A26C39950558 was successfully registered on SimCloud "TESTCLOUD".
 status: VMC 2895A3C3-9AB7-5DA8-A512-60CB9AC4B762 was successfully registered on SimCloud "TESTCLOUD".
 status: Attribute "collect_from_host" was set to "false". Skipping Host OS performance monitor daemon startup
All VMCs successfully attached to this experiment.
The following ATTACHED VMCs are in this experiment (Cloud TESTCLOUD) :
|name                    |host_count      |pool                    |cloud_hostname                  |cloud_ip         
|simzone_a               |15              |sut                     |simzone_a                       |88.74.107.18     
|simzone_b               |15              |sut                     |simzone_b                       |181.71.88.180    
|simzone_d               |15              |lg                      |simzone_d                       |226.200.218.229  
|simzone_c               |15              |sut                     |simzone_c                       |232.131.71.25    

The VMC object simzone_c, attached to this experiment, has the following attributes (Cloud TESTCLOUD) :
|attribute (VMC object key)               |value                                
|update_attempts                          |312                                  
|update_frequency                         |12                                   

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |4.95M                         
|Redis Version                                        |3.0.6                         
|Uptime (in seconds)                                  |8704416                       
|Total Connections Received                           |377210                        
|Total Commands Processed                             |1222709467                    
|Number of Keys                                       |522                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|Collections                                          |13                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |60                            
|VMs                                                  |0                             
|AIs                                                  |0                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ISSUED                                          |12                            
|VMCs ARRIVED                                         |12                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |8                             
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ISSUED                                         |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |0                             
|VMs ISSUED                                           |0                             
|VMs ARRIVED                                          |0                             
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |0                             
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |0                             
|VMs SLA PROVISIONING VIOLATED                        |0                             
|VMs SLA RUNTIME VIOLATED                             |0                             
|VMs APPLICATION ERRORS                               |0                             
|AI RESERVATIONS                                      |0                             
|AIs ISSUED                                           |0                             
|AIs ARRIVED                                          |0                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |0                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ISSUED                                        |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ISSUED                                        |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ISSUED                                         |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |140                           

 ---------------- VMC-WIDE COUNTERS ----------------
|22FD0315-002B-5D68-8C4F-88C6FE47FD88 (simzone_b) VM RESERVATIONS|0                             
|2895A3C3-9AB7-5DA8-A512-60CB9AC4B762 (simzone_c) VM RESERVATIONS|0                             
|F3070B0B-F942-580F-996F-A26C39950558 (simzone_a) VM RESERVATIONS|0                             
|E65CBBAE-AC7D-5374-83BA-971BC910809C (simzone_d) VM RESERVATIONS|0                             

