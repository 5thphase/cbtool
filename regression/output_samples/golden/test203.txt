 status: VMC 9BE56096-E9C1-5C7F-9EAA-E3AAFCDDE5DA was successfully registered on SimCloud "TESTCLOUD".
 status: VMC 01B2A0AA-6D9F-5B16-8896-FE5EC590AB13 was successfully registered on SimCloud "TESTCLOUD".
 status: VMC 66F252C9-8F8A-588B-9E7F-071D8888B1D2 was successfully registered on SimCloud "TESTCLOUD".
 status: VMC 746D6E66-440F-5C77-83C4-004A230AACAF was successfully registered on SimCloud "TESTCLOUD".
 status: Attribute "collect_from_host" was set to "false". Skipping Host OS performance monitor daemon startup
All VMCs successfully attached to this experiment.
The following ATTACHED VMCs are in this experiment (Cloud TESTCLOUD) :
|name                    |host_count      |pool                    |cloud_hostname                  |cloud_ip         
|simzone_a               |15              |sut                     |simzone_a                       |116.121.148.111  
|simzone_d               |15              |lg                      |simzone_d                       |215.217.252.151  
|simzone_c               |15              |sut                     |simzone_c                       |217.238.33.86    
|simzone_b               |15              |sut                     |simzone_b                       |83.251.53.71     

The VMC object simzone_c, attached to this experiment, has the following attributes (Cloud TESTCLOUD) :
|attribute (VMC object key)               |value                                
|update_attempts                          |312                                  
|update_frequency                         |12                                   

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |15.77M                        
|Redis Version                                        |2.8.6                         
|Uptime (in seconds)                                  |12856051                      
|Total Connections Received                           |548495                        
|Total Commands Processed                             |42311721                      
|Number of Keys                                       |506                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|Collections                                          |26                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |60                            
|VMs                                                  |0                             
|AIs                                                  |0                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ISSUED                                          |12                            
|VMCs ARRIVED                                         |12                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |8                             
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ISSUED                                         |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |0                             
|VMs ISSUED                                           |0                             
|VMs ARRIVED                                          |0                             
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |0                             
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |0                             
|VMs SLA PROVISIONING VIOLATED                        |0                             
|VMs SLA RUNTIME VIOLATED                             |0                             
|VMs APPLICATION ERRORS                               |0                             
|AI RESERVATIONS                                      |0                             
|AIs ISSUED                                           |0                             
|AIs ARRIVED                                          |0                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |0                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ISSUED                                        |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ISSUED                                        |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ISSUED                                         |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |140                           

 ---------------- VMC-WIDE COUNTERS ----------------
|01B2A0AA-6D9F-5B16-8896-FE5EC590AB13 (simzone_a) VM RESERVATIONS|0                             
|746D6E66-440F-5C77-83C4-004A230AACAF (simzone_c) VM RESERVATIONS|0                             
|9BE56096-E9C1-5C7F-9EAA-E3AAFCDDE5DA (simzone_d) VM RESERVATIONS|0                             
|66F252C9-8F8A-588B-9E7F-071D8888B1D2 (simzone_b) VM RESERVATIONS|0                             

